sygnal.yaml.sample

```diff
@@ -3,6 +3,21 @@
 # See: matrix.org
 ##
 
+## Database #
```

Comment:
could you make this follow the synapse conventions for sample configs? I feel like it's been serving us well there.

Reply:
Have copied across